Formula/safe-rm.rb

```diff
@@ -4,7 +4,7 @@ class SafeRm < Formula
   url "https://launchpad.net/safe-rm/trunk/0.13/+download/safe-rm-0.13.tar.gz"
   sha256 "f0f792faa076fbf3dbffbb4bc8918fd6bf699775417fef3f5f408f0c69200c12"
   license "GPL-3.0-or-later"
-  head "https://repo.or.cz/safe-rm.git"
+  head "https://git.launchpad.net/safe-rm"
```

Comment:
`head "https://git.launchpad.net/safe-rm.git"`?

Reply:
No, the correct URL is the one without the `.git` for some reason. If I try cloning `https://code.launchpad.net/safe-rm.git`, it asks for credentials.